content/en/tutorials/entity-picking.md

```diff
@@ -41,15 +45,36 @@ PickerRaycast.prototype.onSelect = function (e) {
 
 ## Frame Buffer Picking
 
-Frame buffer based picking uses the [pc.Picker][3] class to render the scene to a internal buffer. When the mouse is clicked the color of the buffer at the mouse location is used to determine which mesh instance was selected. This has some advantages and disadvantages over the collision based method. Advantages include: using a rectangle to pick many items in a scene at once, this method doesn't have require the physics library to be included and active in your project. The main disadvantage is that this uses the `readPixels` method which stalls the graphics pipeline. This can have serious rendering performance implications particularly on mobile.
+Frame buffer based picking uses the [pc.Picker][3] class to render the scene to a internal buffer. When the mouse is clicked the color of the buffer at the mouse location is used to determine which mesh instance was selected. This has some advantages and disadvantages over the collision based method.
+
+Advantages include:
+
+* Able to use a rectangle to pick many items in a scene at once
+* Doesn't require the physics library to be included which reduces preload time.
+
+The main disadvantage is that this uses the `readPixels` method which stalls the graphics pipeline. This can have serious rendering performance implications particularly on mobile.
+
+However, you are able modify the size of the buffer to be smaller to improve the performance at the cost of accuracy. In the example script below, the attribute `pickAreaScale` is used to do this where the lower the number, the faster and less accurate the picker becomes.
```

Comment:
The "however" sounds like you might be able to decrease the impact of the stall, which is not the case. It only speeds up the reading of the texture (back from GPU, and scanning pixels).

But maybe that's too much detail? :)

Reply:
Maybe too much detail 😅

Reply:
maybe just remove the "However" word, which seems to connect this sentence to the previous one?

Reply:
```suggestion
You are able modify the size of the buffer to be smaller to improve the performance at the cost of accuracy. In the example script below, the attribute `pickAreaScale` is used to do this where the lower the number, the faster and less accurate the picker becomes.
```